src/app/debug/table/table-settings-modal/table-settings-modal.component.html

```diff
@@ -10,6 +10,20 @@ <h5 class="modal-title" id="settingsModal">Options</h5>
   <form [formGroup]="settingsForm">
 
     <div class="modal-body container">
+      <div>
+        <h5>Info</h5>
+        <div class="d-flex flex-column gap-2">
+          <div class="d-flex flex-row gap-1">
+            <p class="m-0">Backend Version: {{ backendVersion }}</p>
+            <i class="bi bi-copy cursor-pointer" [appCopyTooltip]="backendVersion"></i>
+          </div>
+          <div class="d-flex flex-row gap-1">
+            <p class="m-0">Frontend Version: {{ frontendVersion }}</p>
+            <i class="bi bi-copy cursor-pointer" [appCopyTooltip]="frontendVersion"></i>
```

Comment:
What if any of these values are still undefined because the http request is slow or because the package.json couldn't be found?
This will cause errors that might break rendering the whole component template

Reply:
But Angular handles undefined strings just fine, they are just treated as an empty string.

Reply:
Okay crazy, that's relatively new because it sure didn't used to be like that
My excuses